akka-actor-tests/src/test/java/akka/event/LoggingAdapterTest.java

```diff
@@ -128,7 +128,7 @@ protected Void match(Object event) {
                     LogEvent log = (LogEvent) event;
                     assertEquals(message, log.message());
                     assertEquals(level, log.level());
-                    assertEquals(mdc, log.mdc().toString());
+                    assertEquals(mdc, log.getMDC().toString());
```

Comment:
hm, what is the purpose of this `expectLog` ?
`String mdc` vs `Map.toString`?
The order of the entries of a Map is undefined.

What is the purpose of `LogJavaTestKit`? The methods have package private visibility. They should at least be protected if it is user api.